certify/dns_certify.ml

```diff
@@ -83,26 +126,35 @@ let pp_q_err ppf = function
   | `Unexpected_reply r -> Fmt.pf ppf "unexpected reply %a" Packet.pp_reply r
   | `No_tlsa -> Fmt.pf ppf "No TLSA record found"
 
+let tlsas_to_certchain public_key tlsas =
+  let certificates, ca_certificates =
+    Rr_map.Tlsa_set.fold (fun tlsa (certs, cacerts as acc) ->
+        if is_certificate tlsa || is_ca_certificate tlsa then
+          match X509.Certificate.decode_der tlsa.Tlsa.data with
+          | Ok cert ->
+            if is_certificate tlsa then
+              (cert :: certs, cacerts)
+            else (* must be is_ca_certificate *)
+              (certs, cert :: cacerts)
+          | _ -> acc
```

Comment:
here we ignore the parsing `Error _` ?

Reply:
yes, this is a shortcoming of the code (but the same behaviour as previously). maybe best to emit a log message..